test/parallel/test-https-connect-address-family.js

```diff
@@ -36,8 +36,13 @@ function runTest() {
 }
 
 dns.lookup('localhost', {family: 6, all: true}, (err, addresses) => {
-  if (err)
+  if (err) {
+    if (err.code === 'ENOTFOUND') {
+      common.skip('localhost does not resolve to ::1');
```

Comment:
In my Ubuntu machines, I had to manually change the `/etc/hosts` to make these tests pass. Perhaps if `localhost` fails, we should give it another try with `ipv6-localhost` instead of skipping.